sdcm/cluster.py

```diff
@@ -1413,6 +1484,16 @@ def get_node_database_errors(self):
                 errors.append({node.name: node_errors})
         return errors
 
+    def set_tc(self, node, dst_nodes):
+        node.remoter.run("sudo modprobe sch_netem")
+        node.remoter.run("sudo tc qdisc del dev eth0 root", ignore_status=True)
+        node.remoter.run("sudo tc qdisc add dev eth0 handle 1: root prio")
+        for dst in dst_nodes:
+            node.remoter.run("sudo tc filter add dev eth0 protocol ip parent 1:0 prio 1 u32 match ip dst {} flowid 2:1".format(dst.private_ip_address))
+        node.remoter.run("sudo tc qdisc add dev eth0 parent 1:1 handle 2:1 netem delay 100ms 20ms 25% reorder 5% 25% loss random 5% 25%")
```

Comment:
let's have the delays and loss percentage as parameters.
In case we would like to run the tests with other scenarios.
We can dedicate a section for 'tc' in the yaml file.